packages/webapp/src/components/Animals/FixedHeaderContainer/index.tsx

```diff
@@ -0,0 +1,106 @@
+/*
+ *  Copyright 2024 LiteFarm.org
+ *  This file is part of LiteFarm.
+ *
+ *  LiteFarm is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  LiteFarm is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details, see <https://www.gnu.org/licenses/>.
+ */
+
+import React, { ReactElement, ReactNode, useLayoutEffect, useMemo, useRef, useState } from 'react';
+import { Paper, useMediaQuery } from '@mui/material';
+import { useTheme } from '@mui/styles';
+import { sumObjectValues } from '../../../util';
+import styles from './styles.module.scss';
+
+const globalNavbarHeight = {
```

Comment:
These aren't dynamic, right? They could be uppercase (or enum)?

Reply:
I changed constants to uppercase! Personally, I've been using camelCase for object and array constants since I found information that uppercase is not always used for non-primitive types. It's nice to make it consistent, but it looks like uppercase, camelCase and snake_case are mixed in our project... 😅

Reply:
Interesting about non-primitive types. I had not thought about it before and it does truly seem to be up to developer  convention. In reading into it, I am horrified to learn that Airbnb style guide recommends not capitalizing even primitive constants if they are not exported 🤣  But the way you did the object after edit seems like a good convention to me (I would have been guilty of even capitalizing even the keys); maybe we can start a mini-LF style guide for this!

Reply:
Oh, I didn't know that! Sounds good!!